### DISCUSSION TITLE
从应用商店一键部署的应用会在应用内部出现网络问题

### DISCUSSION THREAD

**Discussion by yucrushme** [1Panel-dev/1Panel#5089] [Q&A]
### 联系方式

2992576917

### 1Panel 版本

v1.10.7-lts

### 问题描述

从应用商店一键部署的应用会在应用内部出现网络问题，比如dosgame应用里面的所有游戏显示无法下载，koishi会显示插件市场无法连接至网络。我从dockerhub中搜索并部署这两个应用却没有此问题。本人技术水平较低无法进一步排查网络问题。
![屏幕截图 2024-05-21 232516](https://github.com/1Panel-dev/1Panel/assets/77479694/b47acb7d-75cc-4457-adc4-0ab0c5c0448a)
![屏幕截图 2024-05-21 232548](https://github.com/1Panel-dev/1Panel/assets/77479694/9c70f5b2-460a-46e7-974e-90537f3ef0db)

### 重现步骤

可以一直重现，删干净重建也不可以。

### 期待的正确结果

第一次提issue，感谢

**Comment by wanghe-fit2cloud** [comment]
错误原因是因为容器内访问不到外网导致的，可以尝试以下几种方法，看是否生效：

1、关闭服务器防火墙，然后重启网络、重启 docker。
2、进入 `容器` - `配置` 页面，确保 iptables 一定是打开状态。

**Reply by yucrushme** [reply]
感谢，已关闭ufw，并确保容器iptables开启还是无法连上外网😭

**Reply by wanghe-fit2cloud** [reply]
安装应用时，试一下手动删除 `1panel-network` 容器网络，如图：
![image](https://github.com/1Panel-dev/1Panel/assets/46734380/7d267b58-3347-4b72-be64-ea667e97e330)

**Reply by yucrushme** [reply]
非常完美，已经解决，能请教下大佬这是什么原理吗，麻烦大佬指点下😭

**Reply by yucrushme** [reply]
佬，我发现dosgame还是不能连接外网下载游戏，koishi的市场插件可以连接外网更新

**Reply by wanghe-fit2cloud** [reply]
1、可以排查下 1Panel 默认的容器网络是不是和宿主机网络有冲突。
2、进入 `容器` 列表，找到 dosgame 容器，进入 `终端`，通过 ping curl 等命令确认一下容器内部是否能正常访问外网。

**Reply by yucrushme** [reply]
佬，我发现用ping无法解析网址，但是能ping通ip，于是我改了dns让ping通了网址，但是dosgame这个还是不能下载😭
